app/forms/destroy_group_assignments.rb

```diff
@@ -8,6 +8,8 @@ def call
     GroupAssignment
       .where(user_id: user_ids, group_id: group_id)
       .delete_all
+    group = Group.find(group_id)
```

Comment:
Can you extract this into a private method like `destroy_orphaned_group(group_id)` perhaps? Also leave a bit of breathing space between the deletion of groups and the call.